### PR TITLE
Adding Logger Tests

### PR DESCRIPTION
**Pull request by KevinPlus3**
Obtain coverage for `__init__.py` in logger.  Start of adding tests to issue #321.

### CODE REVIEW

File: tests/test_logger.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+from watchmaker.logger import exception_hook
+from watchmaker.logger import LOG_LEVELS
+from watchmaker.logger import make_log_dir
+from watchmaker.logger import prepare_logging
+
+import logging
+import os
+import shutil
+import sys
+import unittest
```
Comment (lorengordon) [inline tests/test_logger.py]: isort order is builtin imports, then third-party imports, then application imports. it's also ordered alphabetically (and case matters). switch this around to:

```
import logging
import os
import shutil
import sys
import unittest

from watchmaker.logger import LOG_LEVELS
from watchmaker.logger import exception_hook
from watchmaker.logger import make_log_dir
from watchmaker.logger import prepare_logging
```

File: tests/test_logger.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import shutil
+import sys
+
+from watchmaker.logger import (LOG_LEVELS, exception_hook, make_log_dir,
+                               prepare_logging)
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
```
Comment (lorengordon) [inline tests/test_logger.py]: How about using `six` for compatibility stuff?

https://pythonhosted.org/six/#unittest-assertions

Comment (KevinPlus3) [inline tests/test_logger.py]: Doesn't seem to support it.  `six` doesn't seem to have a module rename for unittest and the support for assertions is only supported for Python 2.7+, so we would be stuck in the same boat for Python 2.6.  :(

Comment (KevinPlus3) [inline tests/test_logger.py]: And `six` at this moment does not support the `assertLogs` assertion.  No doubt lots of people would love that feature.

Comment (lorengordon) [inline tests/test_logger.py]: Bah.

**Pull request merged**